helmfile.d/values/podsecuritypolicies/common/crossplane.yaml.gotmpl

```diff
@@ -0,0 +1,46 @@
+constraints:
+  crossplane-system:
+    function-capability:
+      podSelectorLabels:
+        pkg.crossplane.io/function: function-capability
+      allow:
+        runAsUser:
+          rule: MustRunAsNonRoot
+      metadata:
+        labels:
+          pkg.crossplane.io/function: function-capability
+      mutation:
+        runAsUser: 65532
```

Comment:
Haven't worked with this before. Could you explain why:
1. You need both `podSelectorLabels` and `metadata.labels`?
 
2. What `allow.runAsUser` does?
    The mutation makes sense, but "allow" sounds like it's bypassing some rule?